packages/server/src/db/linkedRows/LinkController.js

```diff
@@ -143,8 +143,30 @@ class LinkController {
             ? linkDoc.doc2.rowId
             : linkDoc.doc1.rowId
         })
+
+        // if 1:N, ensure that this ID is not already attached to another record
+        const linkedTable = await this._db.get(field.tableId)
+        const linkedSchema = linkedTable.schema[field.fieldName]
+
         // iterate through the link IDs in the row field, see if any don't exist already
         for (let linkId of rowField) {
+          if (linkedSchema.relationshipType === "one-to-many") {
```

Comment:
We should probably store a constant enumeration of these values server-side just so that we can manage it centrally/see what the options could be.